src/main/kotlin/com/hedvig/accountservice/schedulers/web/ScheduleController.kt

```diff
@@ -143,7 +145,8 @@ class ScheduleController @Autowired constructor(
                 ApproveChargeCommand(
                     amount = request.amount,
                     memberId = request.memberId,
-                    approvedBy = approvedBy
+                    approvedBy = approvedBy,
+                    dateApproved = LocalDate.now()
```

Comment:
Since we are operating charges from Stockholm
```suggestion
                    dateApproved = LocalDate.todayInStockholm()
```